docs/running-workflows/supported-systems/snakemake/index.md

```diff
@@ -7,4 +7,13 @@ language, and allows them to be scaled without the need for modifications.
 Snakemake workflows can describe the software required to run them, which is automatically
 deployed in any execution environment.
 
-For additional information, see [Snakemake Documentation](https://snakemake.readthedocs.io).
+For additional information, please see:
+
+- Snakemake documentation: [https://snakemake.readthedocs.io](https://snakemake.readthedocs.io)
```

Comment:
Should we mention the "whether REANA offers any cool novelties such as Snakemake-on-HTC-and-HPC." [bit](https://github.com/reanahub/docs.reana.io/issues/101#issue-1101780844) as part of this task? Or does it belong to a different PR?

Reply:
Good point, it is mentioned in the linked [blog post](https://blog.reana.io/posts/2021/support-for-running-snakemake-workflows/) in `Snakemke in the REANA ecosystem` section (it's a pity it's not possible to link a specific paragraph), but it could be explicitly mentioned here as well

Reply:
It's actually possible, but the hyperlink is not exposed as part of the blog UI.

https://blog.reana.io/posts/2021/support-for-running-snakemake-workflows/#snakemke-in-the-reana-ecosystem